src/components/submission/feed-entry.vue

```diff
@@ -206,7 +208,7 @@ export default {
 .submission-feed-entry {
   .entity-error-message{
     font-size: 12px;
```

Comment:
Since it has more room in the body, maybe it doesn't need to be a smaller font size.